ee/maintained-apps/outputs/brave-browser/windows.json

```diff
@@ -5,10 +5,10 @@
       "queries": {
         "exists": "SELECT 1 FROM programs WHERE name = 'Brave' AND publisher = 'Brave Software Inc';"
       },
-      "installer_url": "https://updates-cdn.bravesoftware.com/build/Brave-Release/x64-rel/win/135.1.77.97/brave_installer-x64.exe",
+      "installer_url": "https://github.com/brave/brave-browser/releases/download/v1.77.97/BraveBrowserStandaloneSilentSetup.exe",
```

Comment:
Looks like this is a more correct download URL that just got added to WinGet. Double-checked since this is moving from a CDN to GitHub, though I haven't validated all (un)install processes.

/cc @jmwatts